doc/python/probe_sample.rst

```diff
@@ -4,6 +4,7 @@ Cable cell probing and sampling
 ===============================
 
 .. module:: arbor
+   :noindex:
```

Comment:
Not sure this should be here? There are currently definitely links to these elements.

Reply:
The module is already indexed in `doc/python/cell.rst`

Reply:
I thought this directive also excludes all that follows. Did you check that it does not break links to stuff in this file?

Reply:
I don't think that this is the case, I do not get warnings about missing links, and the html doc still shows the option to make permalinks for the following entries/sections